app/src/main/java/xyz/whereuat/whereuat/db/entry/ContactEntry.java

```diff
@@ -2,12 +2,26 @@
 
 import android.provider.BaseColumns;
 
+import java.util.AbstractMap.SimpleEntry;
+
 /**
  * Created by kangp3 on 4/5/16.
  */
-public final class ContactEntry implements BaseColumns {
+public final class ContactEntry extends Entry implements BaseColumns {
```

Comment:
This is kind of weird, extending from a class just so you can use some of its static methods. Could you just put `createTable` and `dropTableIfExists` in `WhereuatDbHelper` or something so we don't need a new class? (putting it in `WhereuatDbHelper` might cause a circular import error). Regardless, you don't need to extend `Entry` so I wouldn't.